.ci/conan_center_index.jenkinsfile

```
@@ -0,0 +1,54 @@
+/*
+
+This pipeline is used to generate docker images for ConanCenter index
+
+*/
+
+node('Linux') {
+
+    List<String> conanVersions = ['1.38.0', '1.39.0', '1.40.3']
+    List<String> gccVersions = ['5.5.0', '6.5.0', '7.5.0', '8.4.0', '9.3.0', '10.3.0', '11.1.0']
+    List<String> clangVersions = ['10.0.1', '11.1.0', '12.0.0', '13.0.0']
+    String distro = 'ubuntu'
+    String distro_version = '16.04'
+    String cmake_version = '3.15.7'
+    String python_version = '3.6.15'
+    String jenkins_agent_version = '3.27'
+    // Version of libstdcpp used inside clang images:
+    String libstdcpp_version = '10.3.0'
+    String libstdcpp_patch_version = '28'
+    String libstdcpp_major_version = '10'
+
+    List parameters = [
+        // HEAD from 'master' branch
+        [$class: 'StringParameterValue', name: 'scm_repository', value: 'https://github.com/conan-io/conan-docker-tools'],
+        [$class: 'StringParameterValue', name: 'scm_commit', value: 'master'],
+        [$class: 'StringParameterValue', name: 'scm_base_branch', value: 'master'],
+
+        // [$class: 'StringParameterValue', name: 'suffix', value: 'cci'],
+
+        [$class: 'StringParameterValue', name: 'distro', value: "${distro}${distro_version}"],
+        [$class: 'StringParameterValue', name: 'distro_version', value: distro_version],
+        [$class: 'StringParameterValue', name: 'cmake_version', value: cmake_version],
+        [$class: 'StringParameterValue', name: 'python_version', value: python_version],
+        [$class: 'StringParameterValue', name: 'jenkins_agent_version', value: jenkins_agent_version],
+
+        [$class: 'StringParameterValue', name: 'libstdcpp_version', value: libstdcpp_version],
+        [$class: 'StringParameterValue', name: 'libstdcpp_patch_version', value: libstdcpp_patch_version],
+        [$class: 'StringParameterValue', name: 'libstdcpp_major_version', value: libstdcpp_major_version],
+
+        [$class: 'StringParameterValue', name: 'gcc_versions', value: "${gccVersions.join('\n')}"],
+        [$class: 'StringParameterValue', name: 'clang_versions', value: "${clangVersions.join('\n')}"],
+
+        [$class: 'BooleanParameterValue', name: 'upload_main_images', value: true],
+        [$class: 'BooleanParameterValue', name: 'upload_jenkins_images', value: true],
+        [$class: 'BooleanParameterValue', name: 'upload_latest', value: true],
```

```suggestion
        [$class: 'BooleanParameterValue', name: 'upload_latest', value: false],
```

This needs to be `false`... not all versions can be the latest. We need to handle logic here.